src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/AuditEventRepository.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.licences
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+
+@Repository
+interface AuditEventRepository : JpaRepository<AuditEvent, Long>, JpaSpecificationExecutor<AuditEvent> {
+
+  @Query(
+    nativeQuery = true,
+    value = "select id, timestamp, user, action, details from audit where details ->> 'bookingId' in :bookingIds order by timestamp desc;",
```

Comment:
Question for learning, not a blocker

Given the structure of `details` is a map, is this saying `where the booking ID in the details map is in the list of booking IDs provided`? If so, very cool!